examples/simple_adding.rb

```diff
@@ -0,0 +1,11 @@
+def simple_adding(number)
+  i = 0
+  sum = 0
+  number.times do
```

Comment:
BTW - You will no longer have to increment it, if you use the built in counter. `times` will do that for you, as it goes through each iteration.